README.md

```diff
@@ -219,6 +333,22 @@ When using politeiawww_refclient, the `-use-paywall` flag is true by default. Wh
  * Set the user created in the first refclient execution as admin with politeiawww_dbutil.
  * Run refclient again with the `email` and `password` flags set to the user created in the first refclient execution.
 
+#### Cache
+
+##### Rebuilding the Cache
+
+The cache can be rebuilt using the `--buildcache` flag when starting
```

Comment:
I am for having this knob but politeiad needs to detect this and do the work automatically. This should be a developer knob only.

Reply:
Fixed.  The cache fetches all versions of all proposals from the politeiad inventory and builds the cache from scratch when:

- the cache interface version does not match the version record in the cache
- the cache contains 0 records
- the `--buildcache` flag is used on `politeiad` startup